doc/computation.rst

```diff
@@ -185,6 +185,12 @@ windowed rolling, convolution, short-time FFT, etc.
 
 Because the ``DataArray`` given by ``r.construct('window_dim')`` is a view
 of the original array, it is memory efficient.
+One can also use ``construct`` to compute a weighted mean:
```

Comment:
`weighted rolling`
And nit: we've used `You` or `We` throughout - let's not add another voice

Reply:
Ah, I missed that